packages/office-addin-dev-settings/package.json

```diff
@@ -6,6 +6,9 @@
   "scripts": {
     "build": "rimraf lib && concurrently \"tsc -p tsconfig.json\"",
     "cli": "node lib/cli.js",
+    "lint": "office-addin-lint check",
+    "lint:fix": "office-addin-lint fix",
+    "prettier": "office-addin-lint prettier",
     "test": "mocha -r ts-node/register test/**/*.ts",
     "watch": "rimraf lib && concurrently \"tsc -p tsconfig.json -w\""
```

Comment:
Note that this has "build" and "watch" -- this is probably what you want across the packages...